### ISSUE TITLE
Blockbot command to create a new issue for webgroup, so ideas are dropped in quicker

### ISSUE THREAD

**Issue by JedHazaymeh** [redbrick/blockbot#9]

**Comment by HypnoAnt1** [comment]
A blockbot service account may need to be specified for this to work to be able to submit the issues. The user could be specified based on their discord username.

**Comment by JedHazaymeh** [comment]
> A blockbot service account may need to be specified for this to work to be able to submit the issues. The user could be specified based on their discord username. 

I believe @wizzdom mentioned a RBCmt account, but I think a Webgroup service account would be more suitable (for the name but also for permissions).

**Comment by JedHazaymeh** [comment]
The website has an abstraction layer that uses the Github API to pull Markdown from repos, which I considered extracting into a common resource: useful for cases like this.

The implementation uses Javascript, so it would need to be extracted into a simple REST API (like `github-adapter`, etc) so non-Javascript services (blockbot) can use it.

**Comment by wizzdom** [comment]
> > A blockbot service account may need to be specified for this to work to be able to submit the issues. The user could be specified based on their discord username.
> 
> I believe [@wizzdom](https://github.com/wizzdom) mentioned a RBCmt account, but I think a Webgroup service account would be more suitable (for the name but also for permissions).

Yes, I guess it depends on the use case, we could either use a tightly controlled GitHub token on the RBCmt account or will use a separate webgroup service account. Will have to have a look into github access tokens and how tightly we can limit them in scope.
